packages/hacl-star-raw/hacl-star-raw.0.3.2/opam

```diff
@@ -0,0 +1,35 @@
+x-ci-accept-failures: ["centos-7" "oraclelinux-7"]
```

Comment:
I'm not sure where you go this file from but please keep the comment and `opam-version:` field first.

Reply:
Ah, I'm sorry for the omission. My .opam file in the archive did preserve these comments so it's perhaps opam-publish which reformatted the file? Thanks for adding them back, I'll watch out for it next time.

Reply:
ok i reported it in https://github.com/ocaml-opam/opam-publish/issues/123 let's see if this gets fixed